freqtrade/exchange/exchange.py

```diff
@@ -238,6 +238,19 @@ def get_pair_base_currency(self, pair: str) -> str:
         """
         return self.markets.get(pair, {}).get('base', '')
 
+    def market_is_tradable(self, market: Dict[str, Any]) -> bool:
+        """
+        Check if the market symbol is tradable by Freqtrade.
+        By default, checks if it's splittable by `/` and both sides correspond to base / quote
+        """
+        symbol_parts = market['symbol'].split('/')
```

Comment:
what can be checked here instead of this?

my consideration:
* this is checked by default, as now
* for all supported exchanges (or at least for Kraken and FTX, that do have tradable markets that are not pairs BASE/QUOTE) we can remove calling this method from the base class; if we need custom filtering for Kraken and FTX: use that custom filtering, on other criteria;
* for other (non-supported) exchanges this default check will be in effect, filtering out non-pairs by default; we could adjust their custom exchange classes to allow some non-pairs on those exchanges when we'll know it from other users or own experience...

Reply:
Well thats how its with this PR anyway ...
The pairs split with - are futures on ftx which dont work correctly anyway, and also kraken and binance its not an issue ... 

Just saying its a odd limitation ... Not that its a problem necessarily